dbaccess/block.go

```diff
@@ -96,6 +79,9 @@ func FetchBlock(context Context, hash *daghash.Hash) (*util.Block, error) {
 	if err != nil {
 		return nil, err
 	}
+	if blockLocation == nil {
```

Comment:
Isn't it weird that in one place we return nil if not found, and in the other we return error?